src/main/java/com/flightstats/hub/metrics/StatsdReporter.java

```diff
@@ -0,0 +1,91 @@
+package com.flightstats.hub.metrics;
+
+import com.google.inject.Inject;
+import com.timgroup.statsd.Event;
+import com.timgroup.statsd.StatsDClient;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+public class StatsdReporter {
```

Comment:
Waaay earlier, I almost asked why you didn't just call it a plain old `MetricsReporter`. While that would have been an ok thing to do, I think you were right in leaving it more concrete for now.

My idle dream of having more generic, pluggable metrics reporting is pretty far down my list of nice-to-haves and it would introduce more complexity than we need. The fact that I think it'd be easy to convert over if ever we have time probably indicates that you did the right sort and amount of naming and refactoring.

Reply:
👍